Source/DafnyLanguageServer.Test/Synchronization/DiagnosticsTest.cs

```diff
@@ -47,9 +47,9 @@ decreases y
 }".TrimStart();
       var documentItem = CreateTestDocument(source);
       await client.OpenDocumentAndWaitAsync(documentItem, CancellationToken);
-      var diagnostics = await GetLastVerificationDiagnostics(documentItem, CancellationToken);
+      var diagnostics = await GetLastDiagnostics(documentItem, CancellationToken);
       Assert.AreEqual(0, diagnostics.Length);
-      await AssertNoDiagnosticsAreComing();
+      await AssertNoDiagnosticsAreComing(CancellationToken);
```

Comment:
Isn't the cancellation token a protected property of the parent `LanguageServerTestBase`? Couldn't you use it inside `AssertNoDiagnosticsAreComing` instead of providing it from the child?